packages/frint-store/src/combineEpics.spec.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable import/no-extraneous-dependencies, func-names */
+/* global describe, it */
+import { expect } from 'chai';
+import { Subject } from 'rxjs';
+
+import combineEpics from './combineEpics';
+import ActionsObservable from './ActionsObservable';
+
+describe('frint-store › combineEpics', function () {
+  it('triggers epics correct response', function (done) {
+    const pingEpic = function (action$) {
+      return action$
+        .filter(action => action.type === 'PING')
+        .map(() => ({ type: 'PONG' }));
+    };
+
+    const rootEpic = combineEpics(pingEpic);
+
+    const subject$ = new Subject();
+    const actions$ = new ActionsObservable(subject$);
+    const result$ = rootEpic(actions$);
+    const emittedActions = [];
+
+    result$.subscribe((emittedAction) => {
+      emittedActions.push(emittedAction);
+    });
+
+    subject$.next({ type: 'PING' });
+
+    expect(emittedActions).to.deep.equal([
+      { type: 'PONG' }
+    ]);
+    done();
```

Comment:
`done()` needs to go inside the `.subscribe(fn)`. otherwise we risk finishing the test without any assertion.